### PR TITLE
updates test for dvc-studio-client changes

### PR DESCRIPTION
**Pull request by dberenbaum**
Fixes tests after https://github.com/iterative/dvc-studio-client/pull/78

### CODE REVIEW

**Review by shcheklein — APPROVED** [review]

File: pyproject.toml [hunk 1]
```diff
@@ -33,7 +33,7 @@ dynamic = ["version"]
 dependencies = [
     "dvc>=2.58.0",
     "dvc-render>=0.5.0,<1.0",
-    "dvc-studio-client>=0.10.0,<1",
+    "dvc-studio-client>=0.13.0,<1",
```
Comment (dberenbaum) [inline pyproject.toml:36]: Guess we would need a new dvc-studio-client release and to bump the version here

**Comment by dberenbaum** [comment]
@daavoo Anything to add here? Any objections to a `dvc-studio-client` version bump? Should we do 1.0?

**Comment by daavoo** [comment]
> @daavoo Anything to add here?

tests are failing. I think the import is wrong

>  Any objections to a `dvc-studio-client` version bump? Should we do 1.0?

I think this is only used in tests so it should not be really a major version bump, strictly speaking. The "public" methods didn't change

**Comment by dberenbaum** [comment]
> tests are failing. I think the import is wrong

It's because we haven't released those changes yet, so I am pointing to the old version without them.

> I think this is and env var that we just happen to use in tests here, so it should not be really a major version bump, strictly speaking. The "public" methods didn't change

👍

**Comment by daavoo** [comment]
> > tests are failing. I think the import is wrong
> 
> It's because we haven't released those changes yet, so I am pointing to the old version without them.
> 
> > I think this is and env var that we just happen to use in tests here, so it should not be really a major version bump, strictly speaking. The "public" methods didn't change
> 
> 👍
> 
> So any objection to a dvc-studio-client 0.14.0 release?

no objections

File: pyproject.toml (hunk 1 above)
Comment (dberenbaum) [inline pyproject.toml:36]:
```suggestion
    "dvc-studio-client>=0.14.0,<1",
```

**Pull request merged**